deployments/nfd/overlays/node-feature-rules/platform-labeling-rules.yaml

```diff
@@ -0,0 +1,340 @@
+apiVersion: nfd.k8s-sigs.io/v1alpha1
+kind: NodeFeatureRule
+metadata:
+  name: intel-gpu-platform-labeling
+spec:
+  rules:
+  - labels:
+      gpu.intel.com/platform_MAX1xxx.present: "true"
+    labelsTemplate: "gpu.intel.com/platform_MAX1xxx.count={{ len .pci.device }}"
```

Comment:
The intention was to group first gen MAX series products under one label. If we use "MAX_SERIES", it would group all the possible successors under the same label. I'm open to other proposals, but I wouldn't like to combine everything under one name.

Reply:
If successors have the same features, that should be fine.

If they have different features, they should have a different name [1], but IMHO that's a problem for future.

[1] "MAX_SERIES2", "MAX_SERIES_MINI", "MAX_SERIES_MAXIMUS_MAXIMUM", "AND_NOW_FOR_SOMETHING_COMPLETELY_DIFFERENT"

Reply:
Apparently there has been requests to identify nodes with different Flex cards at least. So for Flex I created _FLEX_140 and _FLEX_170. For Max I created _MAX_1100, _MAX_1550 and _MAX_SERIES (=rest of the ids).

Reply:
NAK.  IMHO dropping architecture label is not acceptable.  If per device type labels are needed, use NFD PCI ID label rule.

PCI ID labels have major advantages over branding ones:
* There's no need to update rules for each new HW variant
* It acts as a fallback that users can use until they can update to GPU plugin / operator / NFD rules having a branded (architecture) label.  As users could be use very old k8s & GPU plugin versions, but new HW...
* We want GPU plugin / operator *releases* to have support for HW when that HW comes out, but individual GPU branding names can change up to until they've been publicly announced, and because releases and product announcements happen at different times, there could be releases missing brand names for HW that's out before next release goes out

While PCI IDs are not branding names, Intel documentation lists them, and user can trivially check the correct value from `dmesg` or `lspci` output.